creme/creme_core/tests/gui/test_history.py

```diff
@@ -516,13 +528,66 @@ def test_render_edition_fk01(self):
             self.render_line(hline, user),
         )
 
+        # old value format '["Hayao Miyazaki", ["position", X], ["image", X]]'
+        current_value = hline.value
+        self.remove_id_in_fk_value(hline)
```

Comment:
I added 2 print()
``` 
print(hline.value)
self.remove_id_in_fk_value(hline)
print(hline.value)
```
And I got this result:
``` 
["Hayao Miyazaki", ["position_id", 4], ["image_id", 7]]
['[', '"', 'H', 'a', 'y', 'a', 'o', ' ', 'M', 'i', 'y', 'a', 'z', 'a', 'k', 'i', '"', ',', ' ', '[', '"', 'p', 'o', 's', 'i', 't', 'i', 'o', 'n', '_', 'i', 'd', '"', ',', ' ', '4', ']', ',', ' ', '[', '"', 'i', 'm', 'a', 'g', 'e', '_', 'i', 'd', '"', ',', ' ', '7', ']', ']']
```
I presume that if the test does not fail it's for good reason...

Reply:
Aie thanks ! I always forgot that this field is a string, sorry. I modify it